### ISSUE TITLE
rowSpan doesn't support columns and table

### ISSUE THREAD

**Issue by Shawnfoto** [bpampuch/pdfmake#2426]
```
var dd = {
    content: [
        {
            table: {
                headerRows:2,
                body: [
                    [{rowSpan: 2,
                         columns: [
                            {
                                  // auto-sized columns have their widths based on their content
                                  width: 'auto',
                                  stack: [{
                                    text: 'From:'    
                                  },
                                  {
                                    text: '\n\n\n'    
                                  },
                                
                                  {
                                    text: 'Contact:'    
                                },
                                {
                                    text: 'Phone:'    
                                },
                                {
                                    text: 'Email(s):'    
                                },
                                {
                                    text: '\n\n'    
                                  },
                                 
                                {
                                    text: 'Docker Name:'    
                                },
                                {
                                    text: 'Limited Access:'    
                                },
                                {
                                    text: 'Docker Open:'    
                                },
                                {
                                    text: 'Docker Close:'    
                                },
                              ],
                              
                            },
                            {
                              width: '*',
                              stack: [
                                 
                                  {text: 'Mine Site'},
                                  {text: '09:00:00'},
                                  {text: '17:00:00'},
                              ],
                              
                            },
                          ],
                    }, {text: 'Properties', colSpan: 3}, '', '', {text: 'Status', rowSpan: 2}],
                    ['', {text: 'A'}, {text: 'B'}, {text: 'C'}, ''],
                    ['Thing', 'Prop A', 'Prop B', 'Prop C', 'Free'],
                ]
            }
        },
    ],
};
```

**Comment by mjomble** [comment]
The same issue was reported in #2069 with a simpler example

**Comment by fe-cj** [comment]
This issue has been resolved with the merge of [PR #2764](https://github.com/bpampuch/pdfmake/pull/2764), and the fix is available starting from versions  [0.3.0-beta.9](https://github.com/bpampuch/pdfmake/releases/tag/0.3.0-beta.9)  and [0.2.11](https://github.com/bpampuch/pdfmake/releases/tag/0.2.11). So we can close this one.
